Content.Server/Shuttles/EntitySystems/ThrusterSystem.cs

```diff
@@ -118,7 +118,8 @@ private void OnTileChange(object? sender, TileChangedEventArgs e)
 
                     foreach (var ent in _mapManager.GetGrid(e.NewTile.GridIndex).GetAnchoredEntities(checkPos))
                     {
-                        if (!EntityManager.TryGetComponent(ent, out ThrusterComponent? thruster) || thruster.Type == ThrusterType.Angular) continue;
+                        if (!EntityManager.TryGetComponent(ent, out ThrusterComponent? thruster) || thruster.Type == ThrusterType.Angular ||
+                            !thruster.RequireSpace) continue;
```

Comment:
Realistically these angular checks can be replaced and just use with RequireSpace.